spanner_orm/tests/migrations_for_emulator_test/create_custom_length_field_f959b767457d.py

```diff
@@ -0,0 +1,32 @@
+"""Spanner ORM migration: create_custom_length_field.
+
+Migration ID: 'f959b767457d'
+Created: 2022-09-13 13:28:34-07:00
+"""
+
+import spanner_orm
+
+migration_id = 'f959b767457d'
+prev_migration_id = '69a8f072dacf'
+
+
+class OriginalTeeTable(spanner_orm.model.Model):
+  """ORM Model with the original schema for the Commands table.
+  Don't update this model, create new migrations instead.
+  """
+
+  __table__ = 'Tee'
+  id = spanner_orm.Field(spanner_orm.String, primary_key=True)
+  cus_str = spanner_orm.Field(spanner_orm.String, length=555)
+  cus_bytes = spanner_orm.Field(spanner_orm.BytesBase64, length=12)
+  cus_strarr = spanner_orm.Field(spanner_orm.StringArray, length=24)
+
+
+def upgrade() -> spanner_orm.CreateTable:
```

Comment:
Should there also be tests for altering the length of an existing column? Does Cloud Spanner even allow that?

Reply:
I intended not to support altering the length of an existing column in this PR and only allow defining it when the table is initialized. Cloud Spanner allows alter the length of an existing column only when it is not a primary key. If it's a primary key column, the update will fail.

Reply:
Ok, that's fine.